packages/aws-cdk-lib/aws-dynamodb/lib/table-v2.ts

```diff
@@ -485,9 +525,15 @@ export class TableV2 extends TableBaseV2 {
       this.addKey(props.sortKey, RANGE_KEY_TYPE);
     }
 
-    this.billingMode = props.billing?.mode ?? BillingMode.PAY_PER_REQUEST;
-    this.readProvisioning = props.billing?._renderReadCapacity();
-    this.writeProvisioning = props.billing?._renderWriteCapacity();
+    if (props.billing?.mode === BillingMode.PAY_PER_REQUEST || props.billing?.mode === undefined) {
```

Comment:
To understand it better, is it intended to set billing mode to `PAY_PER_REQUEST` in case of an undefined value?

Reply:
It's intended yes. PAY_PER_REQUEST is known as OnDemand mode and is the default capacity mode for DynamoDB.